src/components/cards/remote-cards/RemoteCard.tsx

```diff
@@ -58,18 +58,19 @@ const getColorFromString = (color: string | undefined | null) => {
 };
 
 type RemoteCardProps = {
-  card: TrimmedCard;
-  cards: TrimmedCard[];
+  id: string;
   gutterSize: number;
-  carouselRef: React.RefObject<FlashList<TrimmedCard>> | null;
+  carouselRef: React.RefObject<FlashList<string>> | null;
 };
 
-export const RemoteCard: React.FC<RemoteCardProps> = ({ card = {} as TrimmedCard, cards, gutterSize, carouselRef }) => {
+export const RemoteCard: React.FC<RemoteCardProps> = ({ id, gutterSize, carouselRef }) => {
   const { isDarkMode } = useTheme();
   const { navigate } = useNavigation();
   const { language } = useAccountSettings();
   const { width } = useDimensions();
-  const { dismissCard } = useRemoteCardContext();
+  const cards = remoteCardsStore(state => state.cards);
+  const card = remoteCardsStore(state => state.getCard(id)) ?? ({} as TrimmedCard);
+  const dismissCard = remoteCardsStore(state => state.dismissCard);
```

Comment:
can we keep the pattern of prefixing hooks with `use`?

Reply:
the issue with this is it's not technically a hook under the hood since it can be used reactively as well as functionally.

e.g. it can be used like so: `remoteCardsStore.getState()` or `remoteCardsStore.setState({ ... })`.

Reply:
Dunno.. I didn't add that 🤷🏼‍♂️ I find it weird to prefix these stores with `use` personally.